pkg/vulnsrc/ubuntu/ubuntu.go

```diff
@@ -170,8 +170,12 @@ func defaultPut(dbc db.Operation, tx *bolt.Tx, advisory interface{}) error {
 			}
 
 			adv := types.Advisory{}
-			if status.Status == "released" {
+			normalised_status := StatusFromUbuntuStatus(status.Status)
```

Comment:
https://go.dev/doc/effective_go#mixed-caps
```suggestion
			normalisedStatus := StatusFromUbuntuStatus(status.Status)
```

Reply:
also you need to update this variable below :)

Reply:
Yes, need to fix